client/src/components/add-job-post-component/job-info-form/index.jsx

```diff
@@ -0,0 +1,243 @@
+      <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+        <InputLabel id="maxCompensationLabel">Maximum Compensation</InputLabel>
+        <OutlinedInput
+          id="maxCompensationLabel"
+          type="number"
+          inputProps={{ min: 0 }}
+          startAdornment={<InputAdornment position="start">$</InputAdornment>}
+          label="Maximum Compensation"
+          value={jobPostData.maxCompensation}
+          onChange={(e) => {
+            const { value } = e.target;
+            if (/^\d*\.?\d*$/.test(value))
+              handleInputChange(value, "maxCompensation");
+          }}
+          required
+          InputLabelProps={{ required: false }}
+        />
+      </FormControl>
+
+      {/* Compensation Rate Field */}
+      <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+        <InputLabel id="compensationRateLabel">Compensation rate</InputLabel>
+        <Select
+          sx={{ textAlign: "left" }}
+          labelId="compensationRateLabel"
+          id="compensationRateLabel"
+          value={jobPostData.compensationRate}
+          label="Compensation rate"
+          onChange={(e) =>
+            handleInputChange(e.target.value, "compensationRate")
+          }
+          required
+          InputLabelProps={{ required: false }}
+        >
+          {COMPENSATION_RATES.map((jobType) => (
+            <MenuItem key={jobType} value={jobType}>
+              {jobType}
+            </MenuItem>
+          ))}
+        </Select>
+        <FormHelperText>*Required</FormHelperText>
+      </FormControl>
+
+      {/* Hours Per Week Field */}
+      <TextField
+        fullWidth
+        sx={{ m: 1, width: "47%" }}
+        id="hoursPerWeek"
+        type="number"
+        label="Hours per week"
+        inputProps={{ min: 0 }}
+        value={jobPostData.hoursPerWeek}
+        onChange={(e) => {
+          const { value } = e.target;
+          if (/^\d*\.?\d*$/.test(value))
+            handleInputChange(value, "hoursPerWeek");
+        }}
+        required
+        InputLabelProps={{ required: false }}
+      />
+
+      {/* Employment Type Field */}
+      <FormControl fullWidth sx={{ m: 1, width: "96%" }}>
+        <InputLabel>Employment Type</InputLabel>
+        <Select
+          sx={{ textAlign: "left" }}
+          labelId="employmentTypeLabel"
+          id="employmentTypeLabel"
+          value={jobPostData.employmentType}
+          label="Employment Type"
+          onChange={(e) => handleInputChange(e.target.value, "employmentType")}
+          required
+          InputLabelProps={{ required: false }}
+        >
+          {JOB_TYPES.map((jobType) => (
+            <MenuItem key={jobType} value={jobType}>
+              {jobType}
+            </MenuItem>
+          ))}
+        </Select>
+        <FormHelperText>*Required</FormHelperText>
+      </FormControl>
+
+      {/* Creation Date Picker */}
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
```

Comment:
(Differing from the designs, but for the model's sake) We actually are going to make the Creation Date fixed (ie. the text should be greyed out, and the user should not be able to change it. The field should be populated with today's date.